docs/uma-tokenholders/ulabs-liquiditymining.md

```diff
@@ -0,0 +1,23 @@
+---
+title: uLABs Liquidity Mining Program
+sidebar_label: Liquidity Mining
+---
+
+The Liquidity Mining program was [moved under](https://twitter.com/UMAprotocol/status/1324199135080148993) the umbrella of the developer mining, with uLABs as the developer responsible for distributing rewards accrued by 
```

Comment:
Back to my earlier comment, we could do a better job and defining what our liquidity mining program is. Saying it's an umbrella o dev mining doesn't help someone new